src/components/Dialog.css

```diff
@@ -30,3 +30,25 @@
     padding-right: 64px;
   }
 }
+
+.ix-pagination {
+  margin-left: 32px;
+  margin-top: 32px;
+  grid-row: 4;
+}
+
+.ix-next-page-button {
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+}
+
+.ix-chevron-right {
+  padding-top: 5px;
+  padding-left: 3px;
+}
+
+.Button__Button___1ZfFj
+  Button__Button--muted___2Wair
+  Button__Button--small___3yyrk {
+  width: 120px;
+}
```

Comment:
A little worried these class-names could change at build time, IDK how this lib generates them. But maybe not worth worrying about for now.